src/ot_orchestration/dags/config/platform.yaml

```diff
@@ -0,0 +1,102 @@
+gcs_url: gs://open-targets-pre-data-releases/24.06dev-test-config
+
+pis_version: 24.6.0
+etl_version: 24.1.0
+
+chembl_version: '34'
+efo_version: v3.68.0
+ensembl_version: '112'
+
+is_ppp: false
+
+# At the moment, there is no way to determine the dependencies
```

Comment:
My idea originally was for the DAG to be able to infer them without explicitly describing them. This is possible, because you know the files each PIS step produces (it it stated in [its config](https://github.com/javfg/platform-input-support/blob/master/config.yaml#L21)), and you _should_ know what files does [each ETL step needs](https://github.com/opentargets/platform-etl-backend/blob/master/src/main/resources/reference.conf#L87).

The problem is the config file for the ETL does not have a rigid structure for defining the inputs, so it is very hard to figure them out right now. Maybe with some changes or clever code, but meanwhile I just define them here.

There is a great upside of not defining the step dependencies but having them inferred from the way PIS and ETL work: any change in the deps will be reflected in the DAG automatically! Saving the hassle and mistakes of introducing new deps but forgetting to detail them in this file.

We should try to achieve that.

Reply:
With the current approach, even if you forget to add a dependency to the config - it will still be shown in the DAG as a separate node and correct the dependencies list to fit the node just where is should be. 

The only comment why I do not think the dynamic graph dependency is going to fail are:
1. when you have to inject the step in between the existing config, that can not be generated by ingesting the config (like some generic operator)
2. When dag parsing fails to collocate the two dependencies due to discrepancies in between the step input | outputs. You will have to still know where the step should fill in.
3. In gentropy_etl, even if we structurize the parameters and sort them with regards to inputs and outputs, we will still end up with different parameters in dependent steps:
```yaml
steps: 
    - step1:
       output: gs://bucket/datasets/dataset1
    - step2:
       output: gs://bucket/datasets/dataset2
    - step3:
       input: gs://bucket/datasets
```
This will be effectively not possible to figure out the dependency without strictly linking them together.

lets park this for now and see after the `genetics_etl` integration how to fill in.